tests/scripts/all.sh

```diff
@@ -767,6 +767,15 @@ msg "test: allow SHA1 in certificates by default"
 make test
 if_build_succeeded tests/ssl-opt.sh -f SHA-1
 
+msg "build: default config, MBEDTLS_RSA_NO_CRT, make, gcc"
+cleanup
+cp "$CONFIG_H" "$CONFIG_BAK"
+scripts/config.pl set MBEDTLS_RSA_NO_CRT
+make CC=gcc CFLAGS='-Werror -Wall -Werror -O0'
```

Comment:
Why `-O0`? And `-Werror` is repeated, shouldn't that be `-Werror -Wall -Wextra` or `-Werror -Wall`?

Reply:
Thanks for spotting, that's a copy-paste error; the `! MBEDTLS_SSL_CLI_C` test above does it this way, too. I change it to `-Werror -Wall -Wextra -O0`. Regarding the use of optimization, `all.sh` seems to be inconsistent - I have no strong feelings in one direction or the other.